ui/pages/settings/index.scss

```diff
@@ -258,31 +258,26 @@
   }
 
   &__body {
-    padding: 12px 24px;
+    padding: 24px;
 
     @include screen-sm-min {
-      padding: 12px;
+      padding: 24px;
```

Comment:
This adds extra padding to left and right in mobile view. and the content doesn't look aligned with Search
![Screenshot 2023-08-03 at 2 44 04 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/dd0938da-9508-4a58-a781-644984ef993d)

Reply:
Good catch 👁️  will revert this change until further style modifications are needed,